swedbank-pay-design-guide-jekyll-theme.gemspec

```diff
@@ -1,22 +1,33 @@
 # frozen_string_literal: true
+require 'json'
+
+# Required for bundle install
+gem_version = "0.0.1.gemversionnotset"
+
+json_data = Json.load(File.open("version.json"))
```

Comment:
Codacy found an issue: [Useless assignment to variable - `json_data`.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5853053)